src/utils/components/DescriptionModal/DescriptionModal.tsx

```diff
@@ -6,7 +6,7 @@ import TabModal from '../TabModal/TabModal';
 
 type DescriptionModalProps = {
   obj: K8sResourceCommon;
-  onSubmit: (description: string) => Promise<void>;
```

Comment:
Here I think can remain only void as in this component you don't `await` the promise and don't use the promise result

Reply:
the k8sUpdate function I'm using as `onSubmit`, requires this change since it returns a new VM object, and without it, I get an error from TS about wrong type